docs/data/data-grid/export/export.md

```diff
@@ -492,6 +492,23 @@ If you want to keep the formulas working, you can set the `escapeFormulas` optio
 The clipboard export lets you copy the content of the Data Grid to the clipboard.
 For more information, check the [Clipboard copy](/x/react-data-grid/clipboard/#clipboard-copy) docs.
 
+## Recipes
+
+### Server-side export [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+To export server-side data from the DataGrid, use [`updateRows()`](/x/react-data-grid/row-updates/#the-updaterows-method) to temporarily load all data before calling an export function.
+After the export completes, restore the rows to their original state.
+
+The following example shows how to export CSV data from a server-side data source.
```

Comment:
reading this for the third time now ... it feels like this should be rephrased a bit:

```suggestion
The following example demonstrates how to export data as CSV from a server-side data source.
```